src/test/java/labs/introtoprogramming/lab5/object/Parser/ParseTest.java

```diff
@@ -0,0 +1,24 @@
+package labs.introtoprogramming.lab5.object.Parser;
+
+import org.junit.Test;
+
+public class ParseTest {
+
+
+    @Test
+    public void ParseTest() {
+        try {
+
+                ParseObjFile obj = new ParseObjFile("LibertStatue.obj" );
+                obj.getPolygonObject().toString();
+            } catch (java.io.FileNotFoundException e) {
```

Comment:
just add `throws FileNotFoundException` to method signature. It is okay for tests.